acceptance-tests/src/test/java/tech/pegasys/ethsigner/tests/AcceptanceTestBase.java

```diff
@@ -87,9 +87,5 @@ public static void tearDownBase() {
     if (ethNode != null) {
       ethNode.shutdown();
     }
-
-    if (ethSigner != null) {
```

Comment:
If don't do this though and it's only shutdown on JVM exiting, won't we have multiple EthSigners running when running multiple ATs?